js/stat.js

```diff
@@ -0,0 +1,68 @@
+'use strict';
+
+var CLOUD_WIDTH = 420;
+var CLOUD_HEIGHT = 270;
+var CLOUD_X = 100;
+var CLOUD_Y = 10;
+var GAP = 10;
+var BAR_GAP = 50;
+var BAR_WIDTH = 40;
+var BAR_MAX_HEIGHT = 150;
+var TEXT_HEIGHT = 20;
+
+
+function renderCloud(ctx, x, y, color) {
+  ctx.fillStyle = color;
+  ctx.fillRect(x, y, CLOUD_WIDTH, CLOUD_HEIGHT);
+}
+
+function getMaxElement(arr) {
+  var maxElement = arr[0];
+
+  for (var i = 0; i < arr.length; i++) {
+    if (arr[i] > maxElement) {
+      maxElement = arr[i];
+    }
+  }
+
+  return maxElement;
+}
+
+window.renderStatistics = function (ctx, players, times) {
+  var maxTime = getMaxElement(times);
+
+  renderCloud(ctx, CLOUD_X + 10, CLOUD_Y + 10, 'rgba(0, 0, 0, 0.7)');
+  renderCloud(ctx, CLOUD_X, CLOUD_Y, '#fff');
+
+  ctx.fillStyle = '#000';
+  ctx.font = '16px PT Mono';
+  ctx.fillText('Ура вы победили!', CLOUD_X + 40, CLOUD_Y + 30);
+  ctx.fillText('Список результатов:', CLOUD_X + 40, CLOUD_Y + 30 + TEXT_HEIGHT);
+
+  for (var i = 0; i < players.length; i++) {
+    var player = players[i];
+    var time = times[i];
+    var barHeight = (BAR_MAX_HEIGHT / 100) * (time / maxTime * 100);
```

Comment:
по критериям избыточности подходящего критерия на лишние операции нет, но все там, где можно что-то опустить, лучше не вычислять
```suggestion
    var barHeight = BAR_MAX_HEIGHT * time / maxTime
```